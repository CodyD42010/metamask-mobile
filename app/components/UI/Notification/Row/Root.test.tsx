import React from 'react';
import { Linking } from 'react-native';
import { fireEvent } from '@testing-library/react-native';

import renderWithProvider from '../../../../util/test/renderWithProvider';

import NotificationRoot from './Root';
import { ACTIONS, PREFIXES } from '../../../../constants/deeplinks';

const children = <></>;
const styles = {
  wrapper: {},
  trashIconContainer: {},
};
const onDismiss = jest.fn();
const handleOnPress = jest.fn();

describe('NotificationRoot', () => {


  it('renders correctly', () => {
    const { toJSON } = renderWithProvider(
      <NotificationRoot handleOnPress={handleOnPress} onDismiss={onDismiss} styles={styles} children={children}/>,
    );
    expect(toJSON()).toMatchSnapshot();
  });

});
