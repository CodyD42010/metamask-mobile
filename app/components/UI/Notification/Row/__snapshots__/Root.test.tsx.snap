// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationRoot renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Object {
        "height": undefined,
        "opacity": 1,
        "paddingVertical": 10,
      },
    ]
  }
>
  <View
    collapsable={false}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onPress={[MockFunction]}
    style={
      Array [
        Object {
          "transform": Array [
            Object {
              "translateX": 0,
            },
          ],
        },
      ]
    }
  >
    <TouchableOpacity
      onPress={[MockFunction]}
    />
  </View>
  <View
    style={
      Array [
        Object {},
        Object {
          "opacity": 0,
        },
      ]
    }
  >
    <SvgMock
      color="#24272A"
      height={20}
      name="Trash"
      style={
        Object {
          "height": 20,
          "width": 20,
        }
      }
      width={20}
    />
  </View>
</View>
`;
